pkg/apis/serving/k8s_validation_test.go

```diff
@@ -34,16 +35,9 @@ import (
 
 type configOption func(*config.Config) *config.Config
 
-func withMultiContainer() configOption {
+func withFeature(feature string) configOption {
```

Comment:
```suggestion
func withFeature(feature string) configOption {
```

Maybe `withFeatureEnabled` or expand to `withFeature(feature string,  state Flag)`

Reply:
+1 - I'll go with withFeatureEnabled for now since the only use case is to enable features at this point.